source/reference/aggregation.txt

```diff
@@ -98,6 +98,14 @@ find the total number of documents which have the exact array
 
   count = coll.count({ 'categories' => [ 'Chinese', 'Seafood' ] })
 
+.. note::
+
+   The ``count`` method is not the same as the ``countDocuments``
+   method and may not support the same filters. ``countDocuments`` uses the
+   ``$match`` stage of the aggregation pipeline to filter documents. To learn more
+   about limitations of the ``$match`` stage, see :manual:`$match Restrictions </reference/operator/aggregation/match/#restrictions>`
+   in the MongoDB Server documentation.
```

Comment:
FYI I can't add a source constant to the snooty file without breaking the build – unsure if it's a quirk fo the Ruby driver (I notice that it doesn't have many constants in the snooty file as is).

Reply:
hm not sure why that is. The {+mdb-server+} constant was added in the standardization branch, but I think it's fine to not use the constant here